src/argilla/client/feedback/schemas/questions.py

```diff
@@ -122,6 +122,24 @@ class RatingQuestion(QuestionSchema, LabelMappingMixin):
     type: Literal["rating"] = Field("rating", allow_mutation=False)
     values: List[int] = Field(unique_items=True, min_items=2)
 
+    @validator("values")
+    def check_values(cls, values: List[int]):
```

Comment:
When this is deprecated we can use `conlist` instead, right?

Reply:
Or alternatively something like `values: List[int] = Field(..., unique_items=True, ge=1, le=10, min_items=2)`

Reply:
I guess that the current `validator` is there to ensure consistency with the previous version and display the `DeprecationWarning` 👍🏻

Reply:
Exactly! We can force later this validation also on the client side, instead of just warning a message.